src/Game/UI/Gumps/OptionsGump.cs

```diff
@@ -3986,6 +4007,21 @@ private void Apply()
             }
 
             _currentProfile.CBBlackBGToggled = _customBarsBBG.IsChecked;
+
+            if (_currentProfile.CustomBarType != _customBarType.SelectedIndex)
+            {
+                _currentProfile.CustomBarType = _customBarType.SelectedIndex;
+
+                List<HealthBarGumpCustom> hbgcustom2 = UIManager.Gumps.OfType<HealthBarGumpCustom>().ToList();
+                foreach (HealthBarGumpCustom customhealthbar in hbgcustom2)
+                {
+                    UIManager.Add
```

Comment:
what's happen here if these hp bars are anchored?

Reply:
they get unanchored, same with when you toggle normal <-> custom bar

I dont think theres a way to prevent this when the size changes

but it only happens when you change the type, ie option not changed nothing happens